lib/active_model/serializer/utils.rb

```diff
@@ -23,7 +42,9 @@ def include_args_to_hash(included)
     when Symbol
       { included => {} }
     when Hash
-      included.each_with_object({}) { |(key, value), hash| hash[key] = include_args_to_hash(value) }
```

Comment:
No, there are two conventions.  In car,  see semantic blocks in rubocop or tbr style guide?

B mobile phone

> On Sep 20, 2015, at 12:05 PM, L. Preston Sego III <notifications@github.com> wrote:
> 
> In lib/active_model/serializer/utils.rb:
> 
> > @@ -23,7 +42,9 @@ def include_args_to_hash(included)
> >      when Symbol
> >        { included => {} }
> >      when Hash
> > -      included.each_with_object({}) { |(key, value), hash| hash[key] = include_args_to_hash(value) }
> ?
> there are no rubocop warnings here. Are you saying to have rubocop prefer the do/end usage? and warn against {}
> 
> I think usage should depend on complexity and length of line. :-\
> 
> —
> Reply to this email directly or view it on GitHub.
>